UnityClient/Assets/Scripts/Renderer/GridRenderer.cs

```diff
@@ -50,7 +50,7 @@ public class GridRenderer : MonoBehaviour {
     }
 
     private void LoadGridTexture() {
-        material = new Material(Shader.Find("Unlit/WalkableShader"));
+        material = new Material(Shader.Find("Universal Render Pipeline/2D/Sprite-Unlit-Default"));
```

Comment:
This is just to have a valid shader in the grid - probably temporary until we update WalkableShader

Reply:
Don't worry about that, there's nothing fancy about the walkable shader, I don't even remember what it does btw 😅